onnx/backend/test/case/node/cropandresize.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class CropAndResize(Base):
+    @staticmethod
```

Comment:
Can you add a reference implementation?

Reply:
CropAndResize is now a contrib op in onnxruntime, see the C++ implementation here:
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/contrib_ops/cpu/crop_and_resize.cc
Do you mean adding a python implementation? The code is a bit long and this node cannot be easily composed by existing onnx ops. Take existing onnx RoiAlign op for example, there is no reference implementation either because code is long. 

This is a unit test, so the test example is already good to guard the result. In addition, I provide the link for tensorflow implementation. Then it should be enough.

Reply:
Will add